src/steps/users/delete-contact.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseStep, Field, StepInterface } from '../../core/base-step';
+import { FieldDefinition, RunStepResponse, Step, StepDefinition } from '../../proto/cog_pb';
+
+import { baseOperators } from '../../client/constants/operators';
+import * as util from '@run-crank/utilities';
+
+/**
+ * Note: the class name here becomes this step's stepId.
+ @see BaseStep.getId()
+ */
+export class DeleteContact extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Delete an Iterable Contact';
+
+  protected stepType: StepDefinition.Type = StepDefinition.Type.ACTION;
+
+  protected stepExpression: string = 'delete the (?<email>.+) iterable contact';
+
+  protected expectedFields: Field[] = [{
+    field: 'email',
+    type: FieldDefinition.Type.EMAIL,
+    description: 'the email address of the contact',
```

Comment:
- [ ] Per note above: recommend `Contact's email address` for compactness + sentence case.